upgrade/v17xto180/upgrade.go

```diff
@@ -133,18 +133,8 @@ func upgradeBackups(namespace string, lhClient *lhclientset.Clientset, resourceM
 	}
 
 	for _, b := range backupMap {
-		backupTargetName := types.DefaultBackupTargetName
-		vol, err := lhClient.LonghornV1beta2().Volumes(namespace).Get(context.TODO(), b.Status.VolumeName, metav1.GetOptions{})
-		if err != nil {
-			if !apierrors.IsNotFound(err) {
-				return errors.Wrapf(err, "failed to get volume %v of backup %v", b.Status.VolumeName, b.Name)
-			}
-		} else {
-			if vol.Spec.BackupTargetName != "" {
-				backupTargetName = vol.Spec.BackupTargetName
-			}
-		}
-		b.Labels = addLabel(b.Labels, types.LonghornLabelBackupTarget, backupTargetName)
```

Comment:
Why doesn't the original solution work?

Reply:
Because the 'b.Status.VolumeName' of the failed backup is an empty string and it is not allowed to get a volume CR by an empty string name.

Reply:
What’s the returned error of the Volume GET function if the volume name is empty?

Reply:
> What’s the returned error of the Volume GET function if the volume name is empty?

'resource may not be empty'